lib/bugspots/scanner.rb

```diff
@@ -14,8 +14,9 @@ def self.scan(repo, branch = "master", depth = 500, regex = nil)
     end
 
     walker = Rugged::Walker.new(repo)
-    walker.sorting(Rugged::SORT_TOPO | Rugged::SORT_REVERSE)
```

Comment:
Hmm, are you sure about this one? 

https://github.com/libgit2/rugged#commit-walker